R/age_calculate.R

```diff
@@ -0,0 +1,76 @@
+#' Calculate age between two dates
+#'
+#' @description This function calculates the age between two dates using `lubridate` periods.
+#' `lubridate` duration is measured in seconds whereas period is human interpretable unit and varies depending on the context.
+#' Easily switch between the two with the `date_class` argument. Defaulted as "period".
+#'
+#' @param start A start date (typically date of birth) which must be supplied with \code{Date} or \code{POSIXct}
+#' class. \code{\link[base:as.Date]{as.Date()}},
+#' \code{\link[lubridate:ymd]{lubridate::dmy()}} and
+#' \code{\link[base:as.POSIXlt]{as.POSIXct()}} are examples of functions which
+#' can be used to store dates as an appropriate class.
+#' @param end An end date which must be supplied with \code{Date} or \code{POSIXct} class. Defaulted as Sys.time()
+#' @param units Type of units to be used. Seconds, minutes, hours, days, weeks, months and years are accepted.
+#' @param round_down Should returned ages be rounded down to the nearest whole number. Defaulted as `TRUE`.
+#' @param date_class Type of `lubridate` date class. Accepted arguments are "period" and "duration".
```

Comment:
I'm not sure I understand the purpose of `date_class`. I would expect this function to return a numeric. What is the default return type and how does `date_class` change that?